tests/system/test_agent_keep_alive/test_agent_keep_alive.py

```diff
@@ -0,0 +1,339 @@
+'''
```

Comment:
This test was used at the same time HostMonitor and FileMonitor. This is not a good idea for the following reasons:
- Inefficiency: It is necessary to read temporal files twice
- HostMonitor start method is not necessary, `get_manager_received_keep_alive` and `get_agent_keep_alive`  ensure the required messages have been generated, also, it is easier to solve the `update_position` problem previously mentioned.

I should recommend editing HostMonitor to allow gathering group of matched logs and remove FileMonitor logic or, instead, remote HostMonitor log monitoring and use FileMonitoring instead. This can be done by creating a tool (maybe another HostMonitor method) that gather the logs files from the remote hosts.

Reply:
Done in commit 25fd879